### PR TITLE
Gutter menu fix

### PR DESCRIPTION
**Pull request by bomsy**
Fixes an issue where the context menu list no longer changes even if a breakpoint exists (see screenshot below), meaning you can no longer remove a breakpoint, edit a conditional breakpoint etc using the context menu, once the breakpoint is added.

seems to introduced by
https://github.com/devtools-html/debugger.html/commit/9845435ee24460835732523cf72f93bc5f1b7d3b

### Summary of Changes

* stop checking `column` to show the menu for conditional breakpoints

### CODE REVIEW

**Comment by jasonLaster** [comment]
When is the column undefined?

**Comment by bomsy** [comment]
it is set to `undefined` when a conditional break point is added, it happens because of [line 162](https://github.com/devtools-html/debugger.html/blob/master/src/components/Editor/GutterMenu.js#L162)

See here

File: src/components/Editor/GutterMenu.js [hunk 1]
```diff
@@ -152,8 +152,9 @@ class GutterContextMenuComponent extends Component {
       left: event.x,
       top: event.y
     }).ch;
+
     const breakpoint = nextProps.breakpoints.find(
-      bp => bp.location.line === line && bp.location.column === column
```
[the remark this answers is not in the document]
Comment (bomsy) [reply src/components/Editor/GutterMenu.js]: @darkwing does the context menu for the gutter work for you in master for conditional breakpoints?

[the remark this answers is not in the document]
Comment (darkwing) [reply src/components/Editor/GutterMenu.js]: Good shout; we may need to use the feature flag for the `bp.location.column === column` piece.

[the remark this answers is not in the document]
Comment (bomsy) [reply src/components/Editor/GutterMenu.js]: oh cool .. good idea!

**Pull request merged**